src/pynwb/form/build/map.py

```diff
@@ -200,13 +201,18 @@ def __init__(self, **kwargs):
         self.__carg2spec = dict()
         self.__map_spec(spec)
 
+
+    def hack_get_subspec_values(self, *args, **kwargs):
+        return self.__get_subspec_values(*args, **kwargs)
+
```

Comment:
This exposes __get_subspec_values to subclasses of TypeMap; my suggestion is that this gets improved by promoting to get_subspec_values in a later task